frontend/public/components/app.jsx

```diff
@@ -341,11 +361,9 @@ const PollConsoleUpdates = React.memo(function PollConsoleUpdates() {
   }
 
   if (pluginsChanged && !allPluginEndpointsReady && !isFetchingPluginEndpoints) {
-    const pluginEndpointsReady = pluginsData?.plugins?.map((pluginName) => {
-      return coFetchJSON(
-        `${window.SERVER_FLAGS.basePath}api/plugins/${pluginName}/plugin-manifest.json`,
-      );
-    });
+    const pluginEndpointsReady = pluginsData?.plugins?.map((pluginName) =>
+      fetchPluginManifest(pluginName),
+    );
     Promise.all(pluginEndpointsReady)
```

Comment:
There's an existing problem here where if one of the plugins has an error getting the manifest, we will never prompt when a new plugin is installed. I think we should track this as a separate bug, though.

Reply:
Opened a bug to track: https://issues.redhat.com/browse/OCPBUGS-5826